cli/commands/lib/process.js

```diff
@@ -0,0 +1,23 @@
+const { exec } = require("child_process");
+
+function run(command) {
```

Comment:
Not sure if the `lib` directory should be inside `commands`, because it looks like you can run `clio lib` 😄 What do you think about moving it to `cli/lib`?

Reply:
Oh, sure. I'll move it.